### PR TITLE
[BUGFIX] remove sent embedding 

### PR DESCRIPTION
**Pull request by imgarylai**
## Description ##
I'm sorry. I was not carefully reading the documentation. pooled_out is not sent embedding. it is a tensor of pooled representation of the first tokens. 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [x] remove sent embedding (pooled_output)

### CODE REVIEW

**Comment by mli** [comment]
Job PR-618/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-618/1/index.html

File: scripts/bert/embedding.py [hunk 1]
```diff
@@ -103,22 +103,20 @@ def embedding(self, sentences, oov_way='avg'):
...
-                                                         valid_length.astype('float32'))
-            for token_id, sequence_output, pooled_output in zip(token_ids.asnumpy(),
-                                                                sequence_outputs.asnumpy(),
-                                                                pooled_outputs.asnumpy()):
-                batches.append((token_id, sequence_output, pooled_output))
+            sequence_outputs = self.bert(token_ids, token_types,
+                                         valid_length.astype('float32'))
+            for token_id, sequence_output in zip(token_ids.asnumpy(),
+                                                 sequence_outputs.asnumpy()):
+                batches.append((token_id, sequence_output))
```
Comment (kenjewu) [inline scripts/bert/embedding.py]: Yes, `pooled_outputs` is a tensor of pooled representation of the first tokens. But we have already got `sequence_outputs ` here, can we use some methods to fuse `sequence_outputs` to get the basic sentence representation?

Comment (imgarylai) [reply scripts/bert/embedding.py]: My navie idea is to average all the token embeddings. The performance should be tested, though. 
@szha or should I create another issue for this. This fix pull request can be merged first.

Comment (szha) [reply scripts/bert/embedding.py]: I'd lean towards returning sequence outputs and the let the consumer decide how to pool it. This way, the separation of concern is clear.

File: scripts/bert/embedding.py [hunk 2]
```diff
@@ -85,13 +71,13 @@ def __init__(self, ctx=mx.cpu(), model='bert_12_768_12',
         self.bert, self.vocab = gluonnlp.model.get_model(model,
                                                          dataset_name=dataset_name,
                                                          pretrained=True, ctx=self.ctx,
-                                                         use_pooler=True,
+                                                         use_pooler=False,
                                                          use_decoder=False,
                                                          use_classifier=False)
 
     def embedding(self, sentences, oov_way='avg'):
```
Comment (szha) [inline scripts/bert/embedding.py]: would it be more convenient to offer `__call__` as an alias for this?

Comment (szha) [reply scripts/bert/embedding.py]: `result = bert.embedding(sents, oov_way=args.oov_way)` would become
`result = bert_embedding(sents, oov_way=args.oov_way)`

**Comment by szha** [comment]
looks like the CI was broken because tornado was silently upgraded from 5.1.1 to 6.0.1.

File: scripts/bert/embedding.py (hunk 2 above)
Comment (imgarylai) [reply scripts/bert/embedding.py]: Yeah. Added in 8bdbc62

**Comment by imgarylai** [comment]
Rebased this branch.

**Comment by szha** [comment]
@imgarylai I put the fix in #617. Once it's merged, we will need to rebase this PR.

**Comment by imgarylai** [comment]
> @imgarylai I put the fix in #617. Once it's merged, we will need to rebase this PR.

no problem. I will rebase it again.

**Comment by mli** [comment]
Job PR-618/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-618/4/index.html

**Pull request merged**